index.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import express from 'express';
+import hathorLib from '@hathor/wallet-lib';
+
+import config from './config';
+import Wallet from './wallet';
+import apiDocs from './api-docs';
+
+const humanState = {
+  [Wallet.CLOSED]: 'Closed',
+  [Wallet.CONNECTING]: 'Connecting',
+  [Wallet.SYNCING]: 'Syncing',
+  [Wallet.READY]: 'Ready',
+};
+
+const wallet = new Wallet(config);
+wallet.on('state', (state) => {
+  console.log(`State changed to: ${humanState[state]}`);
+});
+
+wallet.on('new-tx', (tx) => {
+  //console.log('New tx arrived:', tx);
+});
+
+wallet.on('update-tx', (tx) => {
+  //console.log('Tx updated:', tx);
+});
+
+const app = express();
+const walletRouter = express.Router()
+
+app.get('/', (req, res) => {
+  res.send('<html><body><h1>Welcome to Hathor Wallet API!</h1><p>See the <a href="docs/">docs</a></p></body></html>');
+});
+
+app.get('/docs', (req, res) => {
+  res.send(apiDocs);
+});
+
+app.get('/status', (req, res) => {
+  res.send({
+    'statusCode': wallet.state,
+    'statusMessage': humanState[wallet.state],
+    'network': wallet.network,
+    'serverUrl': wallet.server,
+    'serverInfo': wallet.serverInfo,
+  });
+});
+
+walletRouter.use((req, res, next) => {
+  if (wallet.state !== Wallet.READY) {
+    res.send({
+      success: false,
+      errmsg: 'Wallet is not ready.',
+      state: wallet.state,
+    });
+    return;
+  }
+  next();
+});
+
+walletRouter.get('/balance', (req, res) => {
+  const balance = wallet.getHTRBalance();
+  res.send(balance);
+});
+
+walletRouter.get('/address', (req, res) => {
+  let address;
+  if ('mark_as_used' in req.query) {
+    address = wallet.getAddressToUse();
+  } else {
+    address = wallet.getCurrentAddress();
+  }
+  res.send({ address });
+});
+
+walletRouter.get('/tx-history', (req, res) => {
+  res.send(wallet.getTxHistory());
+});
+
+walletRouter.post('/simple-send-tx', (req, res) => {
+  const address = req.body.address;
+  const value = parseInt(req.body.value);
+  const tokenUid = Wallet.HTR_TOKEN;
+  wallet.sendTransaction(address, value, tokenUid).then((response) => {
+    res.send(response);
+  }, (error) => {
+    res.send({success: false, error});
+  });
+});
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use('/0', walletRouter);
```

Comment:
Now I see the `/0/` from docs here but why are you using this?

Reply:
I explained in the other comment.